graf2d/gpadv7/inc/ROOT/RAttrAxis.hxx

```diff
@@ -142,16 +143,17 @@ class RAttrAxis : public RAttrBase {
    RAttrAxis &SetLabelsOffset(const RPadLength &len) { fLabelsOffset = len; return *this; }
    RPadLength GetLabelsOffset() const { return fLabelsOffset; }
 
-   const RAttrText &GetLabelsAttr() const { return fLabelsAttr; }
-   RAttrAxis &SetLabelsAttr(const RAttrText &attr) { fLabelsAttr = attr; return *this; }
-   RAttrText &LabelsAttr() { return fLabelsAttr; }
+   RAttrAxis &SetNoLabels(bool on = true) { fNoLabels = on; return *this; }
```

Comment:
What does `!AttrLabels().empty() && !GetNoLabels()` mean? Why do we have both?

Reply:
`AttrLabels()` are graphical attributes (font/size/color) for drawing labels